kokkos.presets.json

```diff
@@ -0,0 +1,68 @@
+{
+    "version"   : 3,
+    "cmakeMinimumRequired" : {
+        "major" : 3,
+        "minor" : 22,
+        "patch" : 0
+    },
+    "configurePresets" : [
+        {
+            "name"          : "default",
+            "displayName"   : "default",
+            "binaryDir"     : "${sourceDir}/build-with-${presetName}",
+            "cacheVariables" : {
+                "CMAKE_EXPORT_COMPILE_COMMANDS" : {"type" : "BOOL"  , "value" : "ON"},
+                "CMAKE_BUILD_TYPE"              : {"type" : "STRING", "value" : "RELEASE"},
+                "CMAKE_CXX_STANDARD"            : {"type" : "STRING", "value" : "17"},
+                "CMAKE_CXX_EXTENSIONS"          : {"type" : "BOOL"  , "value" : "OFF"},
+                "BUILD_SHARED_LIBS"             : {"type" : "BOOL"  , "value" : "ON"}
+            }
+        },
+        {
+            "name"        : "GNU-OpenMP",
```

Comment:
Why the "GNU-" prefix?

Reply:
I think the rationale is that if you compile with e.g. `clang` it would become `clang-OpenMP`. Fine to do what you prefer though.

Reply:
I'd rather drop it and reintroduce later if necessary.

Reply:
Agreed, and done :heavy_check_mark: